src/main/java/no/unit/nva/model/AdditionalIdentifier.java

```diff
@@ -0,0 +1,65 @@
+package no.unit.nva.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import java.util.Objects;
+import nva.commons.core.JacocoGenerated;
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
+public class AdditionalIdentifier {
+
+    @JsonProperty("source")
+    private String source;
```

Comment:
In a sense, the source is the type, I guess

Reply:
the type word is reserved for the objectType